core/runtime/pom.xml

```diff
@@ -44,6 +44,11 @@
             <artifactId>quarkus-arc</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-netty</artifactId>
```

Comment:
Do we actually need the runtime? Would it be sufficient to only add the deployment module?

We removed `quarkus-netty` since it had some nasty security vulnerabilities. If possible, I would like to not include the runtime.

Reply:
No, I think we still some graal subsutions on `quarkus-netty`.

Reply:
The substitutions are done by the deployment, not the runtime. Could you elaborate what you mean?

Reply:
Hmm. That's... not good... I think for the time being, we have no other chance but to re-introduce netty.